.github/workflows/tests.yml

```diff
@@ -39,6 +39,21 @@ jobs:
       - name: Run tests
         run: python manage.py test --shuffle
 
+  test_testdata:
+    name: Test if testdata is up to date
+
+    runs-on: ubuntu-22.04
+
+    steps:
+      - uses: actions/checkout@v4
+      - uses: ./.github/setup_evap
+        with:
+          start-db: true
+
+      - run: python manage.py reload_testdata --noinput
+      - run: python manage.py dump_testdata
+      - run: git diff --exit-code 
```

Comment:
Please group the steps together into one (or two), and name them appropriately :)